app/controllers/users_controller.rb

```diff
@@ -1,5 +1,7 @@
-class UsersController < ApplicationControler
+class UsersController < ApplicationController
```

Comment:
@michellemhey your #index story will need to fix this as well, so let's stay in touch about the timing of our merges.